### PR TITLE
README.md: improved grammar

### PR DESCRIPTION
**Pull request by xternet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,14 +65,14 @@ Recommended:
 
 * Fast CPU with 4+ cores
 * 16GB+ RAM
-* High Performance SSD with at least 1TB free space
+* High-Performance SSD with at least 1TB of free space
```
Comment (holiman) [inline README.md:68]:
```suggestion
* High-Performance SSD with at least 1TB of free space
```
Looks odd to me with the capitalization of `Performance`. It was there from the beginning, but was wrong there too, so please remove that

File: README.md [hunk 2]
```diff
@@ -102,12 +102,12 @@ the main network, but with play-Ether only.
 $ geth --goerli console
 ```
 
-The `console` subcommand has the exact same meaning as above and they are equally
-useful on the testnet too. Please, see above for their explanations if you've skipped here.
+The `console` subcommand has the same meaning as above and is equally
+useful on the testnet too. Please, see above for their explanations if you've skipped them here.
```
Comment (holiman) [inline README.md:106]:
```suggestion
useful on the testnet too. Please, see above for their explanations if you've skipped them here.
```
I think intent was "if you have skipped over the sections above, down to here"

File: README.md [hunk 3]
```diff
@@ -204,12 +204,12 @@ HTTP based JSON-RPC API options:
   * `--http.addr` HTTP-RPC server listening interface (default: `localhost`)
   * `--http.port` HTTP-RPC server listening port (default: `8545`)
   * `--http.api` API's offered over the HTTP-RPC interface (default: `eth,net,web3`)
-  * `--http.corsdomain` Comma separated list of domains from which to accept cross origin requests (browser enforced)
+  * `--http.corsdomain` Comma separated the list of domains from which to accept cross origin requests (browser enforced)
```
Comment (holiman) [inline README.md:207]: Nope, revert

File: README.md [hunk 4]
```diff
@@ -204,12 +204,12 @@ HTTP based JSON-RPC API options:
...
   * `--ws` Enable the WS-RPC server
   * `--ws.addr` WS-RPC server listening interface (default: `localhost`)
   * `--ws.port` WS-RPC server listening port (default: `8546`)
   * `--ws.api` API's offered over the WS-RPC interface (default: `eth,net,web3`)
-  * `--ws.origins` Origins from which to accept websockets requests
+  * `--ws.origins` Origins from which to accept WebSockets requests
```
Comment (holiman) [inline README.md:212]: Meh, why? I think it was better before

File: README.md [hunk 5]
```diff
@@ -228,7 +228,7 @@ APIs!**
 ### Operating a private network
 
 Maintaining your own private network is more involved as a lot of configurations taken for
-granted in the official networks need to be manually set up.
+granted the official networks need to be manually set up.
```
Comment (holiman) [inline README.md:231]: What? Definitely better as it was before

File: README.md (hunk 1 above)
Comment (xternet) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (xternet) [reply README.md]: Not sure, deleted that.

File: README.md (hunk 3 above)
Comment (xternet) [reply README.md]: Ok.

File: README.md (hunk 5 above)
Comment (xternet) [reply README.md]: Ok.

File: README.md (hunk 4 above)
Comment (xternet) [reply README.md:212]: The actual name and more detectable.

**Pull request merged**